inference/README.md

```diff
@@ -0,0 +1,10 @@
+# Inference
+
+For inference we have provided an [inference script](inference.py). Depending on the type of finetuning performed during training the [inference script](inference.py) takes different arguments.
```

Comment:
Please add the chat completion script as well for the Llama-2-chat models

Reply:
Suggest to reformat this README to kind of cover

We support three methods for inference so far

1- inference.py script provides support for HuggingFace accelerate and PEFT fine-tuned models. For more info refer to doc.

2- vLLM that uses the paged attention idea for low latency. For more info refer to doc

3- HuggingFace Text generation, For more info refer to doc

Reply:
-- --model_name and other args might be better to discuss in inference.py related doc.